Cubical/Structures/AbGroup.agda

```diff
@@ -135,9 +131,9 @@ module AbGroupΣTheory {ℓ} where
     0g (helper2 a i) = 0g a
     _+_ (helper2 a i) = _+_ a
     - helper2 a i = - a
-    isMonoid (isGroup (isAbGroup (helper2 a i))) = η-isMonoid (isMonoid (isGroup (isAbGroup a))) i
-    inverse (isGroup (isAbGroup (helper2 a i))) = inverse (isGroup (isAbGroup a))
-    comm (isAbGroup (helper2 a i)) = comm (isAbGroup a)
+    IsGroup.isMonoid (IsAbGroup.isGroup (isAbGroup (helper2 a i))) = η-isMonoid (isMonoid a) i
```

Comment:
Can't you just write `isMonoid (helper2 a i) = η-isMonoid (isMonoid a) i` ?

Reply:
No, I still have no-eta for isMonoid. Maybe you wanted me to remove that too?

Reply:
My comment wasn't about eta? I just wonder why you need all of the qualified projections when isMonoid from AbGroup should be in scope

Reply:
Ah sorry, misread. I get:

Cannot eliminate type  AbGroup  with projection  isMonoid when checking the clause left hand side isMonoid (helper2 a i)

Reply:
Ok, very strange! Can you please add a "TODO: investigate why we need all of the qualified projections" comment so that I might remember to look into it at some point. I would like the library to be as easy to use as possible and having to write this long chain of projections will never scale to more deeply nested algebraic structures